### PR TITLE
initial database setup with schema.sql

### PR DESCRIPTION
**Pull request by tgrahams101**
schema.sql setup and mysql connection

### CODE REVIEW

**Comment by CaptainNemo7** [comment]
Need to remove the key and password. save and import from config file that github won't upload.

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+CREATE TABLE Users (
+  id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  username varchar(40) UNIQUE NOT NULL,
+  password varchar(64) NOT NULL,
+  salt varchar(64) NOT NULL
+);
+CREATE TABLE Sessions (
+  id int NOT NULL AUTO_INCREMENT,
+  cookie_id varchar(64),
```
Comment (janetlee) [inline database/schema.sql:15]: Why does this have an underscore but userId and photoId do not? Let's just stick to one style here for Id's.

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+USE impulse;
+
+CREATE TABLE Users (
+  id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  username varchar(40) UNIQUE NOT NULL,
+  password varchar(64) NOT NULL,
+  salt varchar(64) NOT NULL
+);
+CREATE TABLE Sessions (
+  id int NOT NULL AUTO_INCREMENT,
```
Comment (janetlee) [inline database/schema.sql:14]: Why is Id not a PK value on this table? InnoDB is probably our MySQL engine and it prefers to have a PK value.

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  cookie_id varchar(64),
+  userId INT,
+  FOREIGN KEY (userId) REFERENCES Users(id)
+);
+CREATE TABLE Favorites (
+  id int NOT NULL AUTO_INCREMENT,
```
Comment (janetlee) [inline database/schema.sql:20]: Same here on this table to get a PK value with id.

**Pull request closed without merging**